### ISSUE TITLE
Lots of stats missing

### ISSUE THREAD

**Issue by voltrevo** [TetrisMetrics/BizhawkMetrics#5]
Just got this working this morning, seems to be working ok, but most of the stats described in the readme aren't displayed. I guess there isn't space for them... is there a way to rotate between different stats displayed or perhaps console output somewhere?

I'm running fceux and ran your script via file -> load lua script.

**Comment by TheGrandmother** [comment]
I don't think this is maintained anymore, and there are a lot of stuff that are semi implemented.
I've forked this repo and started to add/remove some stuff.
My fork is not in much of a better state than this one but feel free to see if it works better.
Haven't pushed the latest changes tho, implemented some support for saving stats to files and such.
https://github.com/TheGrandmother/BizhawkMetrics
Feel free to add isues or make PRs there and I'll try to adress them.

this version does output some of the additional stats into the console (just run fceux from the terminal) when the game ends.

**Comment by joshcough** [comment]
Hi,

I will still maintain it. I've just been extremely busy with work this week.

I will be able to address things this weekend.

**Comment by TheGrandmother** [comment]
Nice!
I'll make a pr with some of my changes!

And btw, you should really remove the ROM file from this repo, It's copyrighted.

**Comment by joshcough** [comment]
Thanks! will do.
